nservicebus/messaging/headers.md

```diff
@@ -89,6 +89,11 @@ Identifier of the conversation that this message is part of. It enables the trac
 
 The first message that is sent in a new flow is automatically assigned a unique `Conversation Id` that is then propagated to all the messages that are subsequently sent, thus forming a _conversation_. Each message that is sent within a conversation also has a `RelatedTo` value that identifies the originating message that caused it to be sent. 
 
+There are scenarios where you want to override the Conversation Id because having a single huge conversation doesn't make sense. This can be done by overriding the header with a new value:
```

Comment:
I'd say its not about a single huge one but rather having a business related id instead to make things like `PlaceOrder` and `CancelOrder` show up in the same conversation (and display nicely in SI) by using the OrderId as the conversation id?

I was planning to pull this more philosophical part out to a dedicated page as part of https://github.com/Particular/docs.particular.net/issues/3311

So perhaps we can shorten this to "Overriding the ConversationId is done setting the conversation id header value when sending a message"

Then I can deal with the full explanation as part of #3311?

Reply:
It can be about a single huge one. Picture a long-running process like a scheduler (or implemented as a "We finished one, send a message to do another") that periodically spawns "child conversations" where currently the entire conversation would span multiple months and thousands of messages.

Reply:
THat is exactly what I meant with huge, its never ending saga with timeouts.